modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -136,6 +132,37 @@ def test_blobFromImage(self):
         normAssert(self, blob, target)
 
 
+    def test_model(self):
+        img_path = self.find_dnn_file("dnn/street.png")
+        weights = self.find_dnn_file("dnn/MobileNetSSD_deploy.caffemodel")
+        config = self.find_dnn_file("dnn/MobileNetSSD_deploy.prototxt")
+        frame = cv.imread(img_path)
+        model = cv.dnn_DetectionModel(weights, config)
+        size = (300, 300)
+        mean = (127.5, 127.5, 127.5)
+        scale = 1.0 / 127.5
+        model.setInputParams(size=size, mean=mean, scale=scale)
+
+        iouDiff = 0.05
+        confThreshold = 0.0001
+        nmsThreshold = 0
+        scoreDiff = 1e-3
+
+        classIds, confidences, boxes = model.detect(frame, confThreshold, nmsThreshold)
+
+        refClassIds = (7, 15)
+        refConfidences = (0.9998, 0.8793)
+        refBoxes = ((329.351, 238.952, 85.334, 102.106),
+                    (101.638, 189.152, 34.217, 138.234))
+
+        normAssertDetections(self, refClassIds, refConfidences, refBoxes,
+                             classIds, confidences, boxes,confThreshold, scoreDiff, iouDiff)
+
+        for box, conf, classId in zip(boxes, confidences, classIds):
+            left, top, width, height = box
+            cv.rectangle(frame, (left, top), (left + width, top + height), (0, 255, 0))
```

Comment:
Can we use just `cv.rectangle(frame, box, (0, 255, 0))`?
source: https://docs.opencv.org/master/d6/d6e/group__imgproc__draw.html#gac865734d137287c0afb7682ff7b3db23

Reply:
It works only on the `master` branch.

Reply:
So we are going to merge this PR to master branch as well.